datumaro/util/meta_file_util.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from collections import OrderedDict
+import json
+import os.path as osp
+
+from datumaro.components.annotation import AnnotationType
+from datumaro.util import find
+
+DATASET_META_FILE = 'dataset_meta.json'
+
+def is_meta_file(path):
+    return osp.splitext(osp.basename(path))[1] == '.json'
+
+def has_meta_file(path):
+    return osp.isfile(osp.join(path, DATASET_META_FILE))
```

Comment:
At least 3 functions in this file duplicate the same logic about file path. Maybe, it should be extracted as a function?